lang_portal_backend/api/views.py

```
@@ -626,3 +635,130 @@ def search_transcripts(request):
+        game = WordMatchingGame.objects.create(
+            user=user,
+            total_questions=num_questions
+        )
+        
+        # Get random words for the game
+        words = list(Word.objects.all().order_by('?')[:num_questions])
+        
+        # Create questions
+        for word in words:
+            # Get 3 random incorrect options
+            incorrect_options = list(Word.objects.exclude(id=word.id)
```

Ruff (0.8.2)

Local variable `incorrect_options` is assigned to but never used

Remove assignment to unused variable `incorrect_options`

(F841)